content/yaml-testing/firebase-test-lab.md

```diff
@@ -21,12 +21,15 @@ Please follow the guides in the [Firebase Test Lab documentation](https://fireba
 
 ## Configure testing in Firebase Test Lab in codemagic.yaml
 
-To access the Firebase project from Codemagic, add the service account JSON key file to your `codemagic.yaml` and the Firebase project name as environment variables. See how to [encrypt environment variables](../building/encrypting).
+To access the Firebase project from Codemagic, add the service account JSON key file and the Firebase project name as environment variables in the **Environment variables** section in Codemagic UI (either in Application/Team variables). Select **secure** to encrypt the values. 
+
+Add the group for the environment variable to `codemagic.yaml` as belows:
 
 ```yaml
 environment:
+  groups:
+    - firebase_credentials # <-- (Includes: GCLOUD_KEY_FILE - service account JSON key file)
   vars:
-    GCLOUD_KEY_FILE: Encrypted(...) # <-- Put your encrypted service account JSON key file here
     FIREBASE_PROJECT: "YOUR_FIREBASE_PROJECT_NAME" # <-- Put your Firebase Project Name here
```

Comment:
the vars section seems unnecessary

Reply:
But this can be added 
FIREBASE_PROJECT: "YOUR_FIREBASE_PROJECT_NAME" # <-- Put your Firebase Project Name here

Reply:
We're recommending saving it to environment variables in the UI at the top of the section, hence my comment.

Reply:
Changes done